website/static/js/markdown.js

```diff
@@ -35,7 +35,7 @@ var mfrURL = window.contextVars.node.urls.mfr;
 var osfURL = window.contextVars.osfURL;
 
 var getMfrUrl = function (guid) {
-    return mfrURL + 'render?url='+ osfURL + guid + '/?action=download%26mode=render';
+    return mfrURL + 'render?url='+ osfURL + guid + '/download/?action=download%26mode=render';
```

Comment:
@Johnetordoff, can you explain why this is necessary for quickfiles?

Reply:
Without a download path the Quickfiles guid will attempt to resolve to the ember app instead of accepting an action parameter.

Reply:
Ah, got it. Thank you for explaining!